gen/definitions/generic/device_list.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: Devices

```

Comment:
We typically use singular for resource names and I would also suggest to name the yaml file accordingly -> device.yaml